torch/csrc/jit/passes/dead_code_elimination.cpp

```diff
@@ -107,6 +108,12 @@ class DeadCodeEliminator {
       if (hasSideEffects(node)) {
         mark(node);
       }
+
```

Comment:
Passes don't typically descend into subgraphs. Is this still needed now that the Function* is actually stored in the type?

Reply:
actually, I think I might be able to remove this particular one, but then Inliner will need to descend into subgraphs and clean up unused `Function` literals